core/src/main/java/nu/yona/server/properties/YonaProperties.java

```diff
@@ -191,4 +194,14 @@ public void setTestServer(boolean isTestServer)
 	{
 		this.isTestServer = isTestServer;
 	}
+
+	public Duration getOverwriteUserConfirmationCodeValidityTime()
```

Comment:
The name seems to overlap with the existing `getPinResetRequestExpirationTime` property. "Valid" and "not expired" seem to mean the same. So shouldn't we just add the rate limiting/throttling and for the rest keep sending, even within 30 seconds?

Reply:
The app has the unpleasant habit of sending requests multiple times in quick succession. Given only a few requests are made, throttling does not address it, but we're sending multiple SMSes (extra cost) and we get concurrency issues.
However, I agree the name isn't great. It sounds like expiration time, but it isn't.  It's the "nonresend" or "je doet het er maar mee" interval. What would be a good name for that?

Reply:
The new name looks great! 😄 

"The app has the unpleasant habit of sending requests multiple times in quick succession." Shouldn't we fix that habit instead? I think that is not unreasonable?

Reply:
Absolutely, but I've asked that before, with no result yet. App development doesn't make much progress at the moment.